clairmeta/dcp_check_am.py

```diff
@@ -41,6 +42,19 @@ def check_am_xml(self, am):
             am['Info']['AssetMap']['Schema'],
             self.dcp.schema)
 
+    def check_am_volume_count(self, am):
+        """ The VolumeCount element shall be 1
+
+            References:
+                SMPTE ST 429-9:2014 5.4
```

Comment:
We tend to use hyphen for the year too: 429-9-2014

Might be parsed at some point later on so better keep it consistent I guess.